cmk/gui/plugins/wato/active_checks.py

```diff
@@ -2490,3 +2490,30 @@ def _valuespec_active_checks_elasticsearch_query():
         name="active_checks:elasticsearch_query",
         valuespec=_valuespec_active_checks_elasticsearch_query,
     ))
+
+def _valuespec_active_checks_crl():
+    return Dictionary(
+        title=_("Check CRL expiry"),
+        help=_("Checks if a Certificate Revocation List is still valid"),
+        optional_keys=[],
+        elements=[
+        ("minutes",
+        Tuple(title=_("Minutes left"),
+              help=_("These levels make the check go warning or critical whenever the "
+                     "remaining validity of the monitored CRL is too low."),
+              elements=[
+                  Integer(title=_("warning at"), unit=u"Minutes", default_value=2880),
```

Comment:
Please use the valuespec `Age` here. In checks/check_crl, the actual value will then be seconds, so you have to transform there to what the Nagios plugin expects.